yaeger.spacesimulator/src/main/java/com/yaeger/spacesimulator/services/ObjectCreationService.java

```diff
@@ -33,9 +33,9 @@ public void addCentrePlanet(ObjectPlacementData data) {
 		CentrePlanet planet = new CentrePlanet(
 			new Coordinate2D(data.getStartPosition()),
 			new Coordinate2D(Point2D.ZERO),
```

Comment:
new Coordinate2D() maakt sws expliciet new Coordinate(0,0), dat is wat je wil met die ZERO denk ik he?